src/main/java/nl/novi/spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,42 @@
+package nl.novi.spring_boot_tech_it_easy_controller.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TelevisionsController {
+
+
+    @GetMapping("/televisions")
+    public ResponseEntity<String> getAllTelevisions() {
+//        response return 200 status with string
+        return ResponseEntity.ok("televisions");
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<String> getTelevisionbyId(@PathVariable int id) {
+//        response return 200 status with specific string id
+        return ResponseEntity.ok("television with id: " + id);
+    }
+
+    @PostMapping("/televisions")
+    public ResponseEntity<String> createTelevision(@RequestBody String television) {
```

Comment:
Er gebeurt nu niks met de RequestBody die je hier ontvangt. Deze had je in een lisjt kunnen opslaan, of kunnen returnen in de response body.